deps.edn

```diff
@@ -1,3 +1,4 @@
-{:deps
+{:paths ["src"]
```

Comment:
This is supposed to be optional, so I'm guessing blob generation isn't factoring in the system deps.edn, which means there's another bug where blob generation fails if it's using dependencies that aren't in your ~/.m2 (and a plethora of additional issues around repos & snapshots).

Reply:
I believe he's saying that `:paths ["src"]` is the default.
What is the issue that made you add this line?

Reply:
Oh, right, I got confused because running it like in the docs via `clj -m unrepl/make-blob` errors and you have to do `clj -m unrepl.make-blob` but I can confirm it works now, even without the source path added.

I still think making that explicit doesn't hurt or do you want me to take it out and force push?

Reply:
You can just add a commit that removes it. I'll squash them on merge.

Reply:
done!